src/js/components/LegislativePriorities.jsx

```diff
@@ -0,0 +1,30 @@
+import useAirtableCMS from "../hooks/useAirtableCMS";
+import Markdown from "react-markdown";
+import remarkGfm from "remark-gfm";
+
+const LegislativePriorities = (props) => {
+  const legislativePriorities = useAirtableCMS({
+    baseID: "app1YqNgXXkVH04nO",
+    tableName: "Legislative Priorities",
+    keyField: "name",
+    fieldMapping: {
+      name: "Name",
+      content: "Content",
+    },
+  });
+
+  return (
+    <div className="component LegislativePriorities">
+      {legislativePriorities.map((record) => {
+        return (
+          <div className="container">
```

Comment:
Elements in an iterator should have a "key" prop for efficient diffing by React (see https://stackoverflow.com/questions/28329382/understanding-unique-keys-for-array-children-in-react-js)

Reply:
Oh yes! missed that. Thanks for pointing out.